app/src/main/java/app/insti/fragment/FeedFragment.java

```diff
@@ -37,6 +38,7 @@
  */
 public class FeedFragment extends BaseFragment {
 
+    String TAG = "FeedFragment";
```

Comment:
Why is this file changed at all?

Reply:
Added logs to check the flow of this fragment. Forgot to remove it. Sorry